packages/integration-karma/test/profiler/sanity/profiler.spec.js

```diff
@@ -2,9 +2,27 @@ import { createElement } from 'lwc';
 import Container from 'x/container';
 
 describe('Profiler Sanity Test', () => {
+    const hasPerfMarksAndMeasures =
+        typeof performance !== 'undefined' && performance.getEntriesByType;
+    let numMarks = 0;
```

Comment:
can you rename this to marks / measures logged before test starts? 

and then just add a comment about why we need this approach

Reply:
Done.